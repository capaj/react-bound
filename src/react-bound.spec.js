import Bound from './react-bound'
import 'jest'
import React from 'react'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import { mount } from 'enzyme'

Enzyme.configure({ adapter: new Adapter() })

describe('react-bound', () => {
  it('renders', async () => {
    const mounted = mount(
      <Bound to={{}}>
        <textarea name="formA.unicornInput" />
        <textarea name="formB.unicornInput" />
      </Bound>
    )
    expect(mounted.html()).toMatchSnapshot()
  })
})
