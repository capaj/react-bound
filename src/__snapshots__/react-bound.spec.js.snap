// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`react-bound renders 1`] = `"<textarea name=\\"formA.unicornInput\\"></textarea>"`;
